geoserver/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# A GeoServer installation
+FROM java:8-alpine
+
+MAINTAINER Azavea
+
+ENV GEOSERVER_VERSION 2.11.2
+ENV GEOSERVER_ZIP_URL http://sourceforge.net/projects/geoserver/files/GeoServer/${GEOSERVER_VERSION}/geoserver-${GEOSERVER_VERSION}-bin.zip
```

Comment:
Files hosted on SF could lead to issues later. I think we should leave things as is for now, but keep an eye on failures/speed to determine if we should put this up somewhere.